third_party/nuclio/main.py

```diff
@@ -75,3 +86,10 @@ def handler(context: Context, event: Event) -> Context.Response:
         content_type="application/json",
         status_code=200,
     )
+
+
+def too_small_to_care(mask: cv2.typing.MatLike, size: tuple) -> bool:
+    image_area = size[0] * size[1]
+    mask_area = cv2.countNonZero(mask)
+    percentage = (mask_area / image_area) * 100
```

Comment:
I'm wondering if normalizing by the `image_area` makes sense...what if the width/height ratio varies?

Reply:
We don't have much variation on that given the controlled environment to collect the  pictures. However, I still fail to understand why the area is not a valid measurement to consider. I mean, why do we care about the width/height ratio?

Reply:
I do have to agree that it's kinda hit-or-miss using this arbitrary area. It was calculated somehow from info from the dataset, however.

Reply:
If you have an image of 1024x1024 and crop it to 1024x768 you will have different normalization factors, although the image quality (e.g. dpi or pixel size) still the same.